composer.json

```diff
@@ -39,8 +39,11 @@
     ],
     "require": {
         "php": "^7.1",
-        "phpspec/phpspec": "^4.2 || ^5.0 || ^6.0",
-        "phpunit/php-code-coverage": "^5.0 || ^6.0 || ^7.0"
+        "phpspec/phpspec": "^5.0 || ^6.0",
+        "phpunit/php-code-coverage": "^6.0 || ^7.0"
+    },
+    "conflict": {
+        "sebastian/comparator": "<2.0"
```

Comment:
I had to change this because of failing low-deps tests caused by this bug on PHP >=7.2: https://github.com/sebastianbergmann/comparator/pull/30

Reply:
@drupol This may also help you with #18.
```
"conflict": {
    "phpunit/php-code-coverage": "<6.0"
}
```

Reply:
Oooooh !!! Trying it out :-) Thanks mate!